app/src/main/java/ch/sdp/vibester/activity/TypingGameActivity.kt

```diff
@@ -108,7 +123,7 @@ class TypingGameActivity : GameActivity() {
                     try {
                         val list = Song.listSong(task.await())
                         for (x: Song in list) {
-                            guess(x, findViewById(R.id.displayGuess), this@TypingGameActivity, gameManager )
+                            guess(x,findViewById(R.id.displayGuess),this@TypingGameActivity,gameManager)
```

Comment:
```suggestion
                            guess(x, findViewById(R.id.displayGuess), this@TypingGameActivity, gameManager)
```
Why having deleted the space. It is with this format everywhere in the code ?

Reply:
I think each parameter was on different line before. When I was moving them into one line, the space got deleted... We should format the code... I can do that in next PR